connector/src/main/java/org/geysermc/connector/network/translators/world/block/entity/PistonBlockEntity.java

```diff
@@ -369,6 +369,11 @@ private void correctPlayerPosition() {
         double delta = Math.abs(progress - lastProgress);
         Vector3d extend = movement.negate().toDouble().mul(delta);
         playerCollision.extend(extend.getX(), extend.getY(), extend.getZ());
+        // Shrink the collision in the other axes slightly, to avoid false positives when pressed up against the side of blocks
```

Comment:
Shouldn't you set this back after?

Reply:
`playerCollision` is a separate object from the bounding box in `CollisionManager`.